django/curator/management/commands/curator_cluster_tags.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+from django.core.management.base import BaseCommand
+
+from curator.tag_deduplication import TagClusterer
+
+
+class Command(BaseCommand):
+    help = "Cluster Tags using dedupe"
```

Comment:
this help method should include the usage details you added to this PR, with clear and consistent language. How and when should this class be used, with example invocations and clear instruction on where things will be persisted, and how to parameterize / change that.